### PR TITLE
Fix filling in the nickname, overriding the username (in add network tab)

### PR DESCRIPTION
**Pull request by PolarizedIons**
Fixes issue #843

### CODE REVIEW

**Comment by MaxLeiter** [comment]
Tests are failing due to some trailing spaces, @PolarizedIons 

https://ci.appveyor.com/project/astorije/lounge/build/1569#L410

**Comment by PolarizedIons** [comment]
Oh whoops, I could have sworn I checked for that. Thank you @MaxLeiter - fixed now

**Comment by PolarizedIons** [comment]
ping?

**Comment by astorije** [comment]
@xPaw, can you think of a less convoluted way to do so? This is a fairly simple feature, we shouldn't need 2 handlers and 20+ lines of code for it IMO :/

**Comment by xPaw** [comment]
@astorije While it does seem silly, the code quite makes sense.

File: client/js/lounge.js [hunk 1]
```diff
@@ -1283,9 +1283,29 @@ $(function() {
 		);
 	});
 
+	forms.on("focusin", ".nick", function() {
+		// Need to set the first "lastvalue", so it can be used in the below function
+		var nick = $(this);
+		if (nick.data("lastvalue") === undefined) {
```
Comment (xPaw) [inline client/js/lounge.js]: What does this check accomplish?

Comment (PolarizedIons) [inline client/js/lounge.js]: I can't seem to find a reason for it either, probably something that got left over after I rewrote it. Thanks for the catch, removed it

**Pull request merged**